### ISSUE TITLE
Setting availability to off when not updating

### ISSUE THREAD

**Issue by nodiaque** [michaelwoods/onstar2mqtt#332]
Hello,

As we know, onstar tend to be a big hit and miss. After a while, you request diagnostic and get nothing. It would be nice if the availability topic would change to false when it didn't update. Also having the last successful update time in a topic would be good.

Thank you

**Comment by nodiaque** [comment]
Oh nice, I'll be happy to try that. How do I switch to your fork?

**Comment by nodiaque** [comment]
just found out I was already on your fork. Starting the testing, I'll report soon

**Comment by nodiaque** [comment]
Ok, I just saw that I need to add a variable for the polling status. My idea was to use the current availability status channel that is under homeassistant/vin/available. Just change this one to false and HomeAssistant (or anything other home automation using this like OpenHab) will automaticly report as offline. Aside of that, I do see a json answer at the path provided saying 
{
"ok" :
{
"message" : "Data polled successfully"
}
}

I'll check once it fail to poll.

**Comment by nodiaque** [comment]
I'm not on HA so I cannot say on that. For me this put the things (in openhab we call them things) to offline. I can still read the last value they add but since it's offline, it tell me it failed last polling. Even having it "unavailable" in HA for me would say "well, onstar doesn't work right now".

**Comment by nodiaque** [comment]
yeah I just saw the lastpoolsuccessful one, I'll incorporate this somewhere.

Is it normal this container doesn't properly stop? At least in unraid, when I request a stop, it wait the full 60s timeout and then kill it

**Comment by tylerpieper** [comment]
@BigThunderSR You could wrap your car info in a conditional card and hide/replace it when the entities go unavailable.
